components/Grid Simplified/grid.css

```diff
@@ -0,0 +1,36 @@
+.grid-btn{
+    padding: 1rem 3rem;
+    background: none;
+    border: none;
+    text-decoration-line: underline;
+    color: var(--text-color);
+    font-size: x-large;
+    font-family: monospace;
+}
+.txt{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.flex-txt{
+    display: flex;
+    flex-direction: column;
+    /* justify-content: center; */
+    align-items: center;
+}
+.grid-content{
+    margin: 2rem;
+    border: 1px solid #797572;
+}
+.grid-box{
+    width: 100%;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    border: 4px solid #6b6a6a;
```

Comment:
@jeetbhalani15 I see that this color is being used repeatedly so use var for that